src/jit/lclvars.cpp

```diff
@@ -2689,6 +2689,10 @@ void Compiler::lvaSortByRefCount()
     lvaTrackedCount             = 0;
     lvaTrackedCountInSizeTUnits = 0;
 
+#ifdef DEBUG
+    VarSetOps::AssignNoCopy(this, lvaTrackedVars, VarSetOps::MakeEmpty(this));
```

Comment:
@BruceForstall - pointed out that lvaTrackedVars is defined only in debug.  In my source search I was also looking at jit32 occurrences.

This fix looks good.